lib/Seq.pm

```diff
@@ -134,15 +134,15 @@ sub annotateFile {
 
   # To avoid the Moose/Mouse accessor penalty, store reference to underlying data
   my $db = $self->{_db};
-  my $refTrackGetter = $self->tracksObj->getRefTrackGetter();
-  my @trackGettersExceptReference = @{$self->tracksObj->getTrackGettersExceptReference()};
+  my $refTrackGetter = $self->{_tracks}->getRefTrackGetter();
```

Comment:
$self->{_tracks} is initialized in the BUILD method, and is done so because tracksObj is lazy and its lazy init method sets the database to read only mode. We want that to happen as early in the program as possible, because otherwise LMDB will be in write mode, and we do not respect write transaction semantics in annotation (it must be read only). This gives us a large throughput benefit.

Reply:
makes sense to me, but could this explanation (and the others on this PR giving context / clarifying intent) go into the code as comments?